go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/CosmWasm/wasmd v0.28.0-osmo-v12
 	github.com/cosmos/cosmos-proto v1.0.0-alpha8
-	github.com/cosmos/cosmos-sdk v0.46.2
+	github.com/cosmos/cosmos-sdk v0.46.3
```

Comment:
are we good with bumping this?

Reply:
This should be redundent right? We're replacing it any ways, i think its just an annoying proto thing